scripts/qesap/qesap.py

```diff
@@ -23,20 +24,6 @@
 log = logging.getLogger('QESAPDEP')
 
 
-def os_path_exists(path):
```

Comment:
This mock was at purpose as indicated in the comment

Reply:
I dropped it because it's unused

Reply:
You are right, it is fine to drop